src/expression.c

```diff
@@ -828,6 +946,24 @@ Expression *resolveUFCS(Scope *sc, CallExp *ce)
                 else
                     return NULL;
             }
+            if (t->ty == Tclass || t->ty == Tstruct)
+            {
+                Expressions results;
+                iterateAliasThis(sc, eleft, &atSubstUnaDotId, (void*)ce, &results);
+                if (results.dim == 1)
+                {
+                    return results[0];
+                }
+                else if (results.dim > 1)
+                {
+                    ce->error("Unable to unambiguously resolve %s Candidates:", ce->toChars());
```

Comment:
Errors shouldn't have any capitals in the message, only symbol names have that privilege. ;)